module/js/integrations/ChrisPremades.js

```diff
@@ -0,0 +1,246 @@
+import {IntegrationBase} from "./IntegrationBase.js";
+import {SharedConsts} from "../../shared/SharedConsts.js";
+
+// Cheat and pretend we're not always overriding, as our patches are temporary/highly contextual
+const _LIBWRAPPER_TYPE_TEMP = "MIXED";
+
+// Cache the original duplicate method
+const _DUPLICATE = duplicate;
+
+/**
+ * Designed for use with `chris-premades` v0.3.7
+ * See: https://github.com/chrisk123999/chris-premades
+ */
+export class IntegrationChrisPremades extends IntegrationBase {
```

Comment:
this is mega-brittle and would be much nicer if CPR exposed an API to let one e.g. grab its data by name, but I'm hesitant to start that "please add an API : )" for obvious reasons